### PR TITLE
Add suffixes to typos from "cons" to "conv"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -13686,13 +13773,20 @@ contiguious->contiguous
 contiguities->continuities
 contiguos->contiguous
 contiguous-non->non-contiguous
+contina->contain
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:13776]: I guess `continua` and `fontina` cheese are rare, aren't they?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -13686,13 +13773,20 @@ contiguious->contiguous
...
+continaed->contained
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:13777]: Any `continued` typos out there?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -13732,14 +13837,33 @@ contniued->continued
 contniues->continues
 contnt->content
 contol->control
+contole->control, console, condole,
+contoled->controlled, condoled,
 contoler->controller
+contolers->controllers
+contoles->controls, consoles, condoles,
+contoling->controlling, condoling,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:13845]: Perhaps add `consoling` as well?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -13732,14 +13837,33 @@ contniued->continued
 contniues->continues
 contnt->content
 contol->control
+contole->control, console, condole,
+contoled->controlled, condoled,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:13841]: Perhaps add `consoled` as well?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -13877,24 +14029,37 @@ convense->convince
...
 convensions->conventions, conversions,
 convential->conventional
+conventience->convenience
+conventiences->conveniences
 conventient->convenient
+conventiently->conveniently
 convenvient->convenient
-conver->convert
-convered->converted, covered,
+conver->convert, cover, convey, confer,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:14037]: Perhaps `convex` too?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -13877,24 +14029,37 @@ convense->convince
...
+converage->coverage, converge,
+converaged->converged
+converagence->convergence
+converagent->convergent
+converages->coverages, converges,
+converaging->converging
+converation->conversation
+converations->conversations
+convered->converted, covered, conveyed,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:14046]:
```suggestion
convered->converged, converted, covered, conveyed,
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -13877,24 +14029,37 @@ convense->convince
...
 convereted->converted
 convergance->convergence
-convering->converting, covering,
+convering->converting, covering, conveying,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:14049]:
```suggestion
convering->converging, converting, covering, conveying,
```

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:13777]: I can't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:13776]: These words are too rare.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**